packages/koa-react-sidecar/example/index.tsx

```diff
@@ -1,11 +1,23 @@
 import React from 'react';
-import {createServer} from '../dist/server';
+import {ApolloProvider} from '@shopify/react-graphql';
+import {createServer} from '../src/server';
 
 function MockApp() {
   return <div>I am react</div>;
 }
 
 createServer({
   port: 4444,
-  render: () => <MockApp />,
+  graphQLClientOptions: {},
+  render: ({graphQLClient}) => {
+    if (graphQLClient) {
+      return (
+        <ApolloProvider client={graphQLClient}>
```

Comment:
I'll elaborate on this more down the line but this feels like it's a bit too in the middle of too opinionated and not opinionated enough. Specifically, it feels weird that we put in a tonne of work to set up the apollo client but then the user still needs to import apollo and render the provider themselves